source/model/Database.py

```diff
@@ -15,13 +15,15 @@ def get_device(self, device_id):
 
```

Comment:
We should discuss on tuesday if we chooses suzannes way of setting the counter or mine(Lars).